### PR TITLE
NETOBSERV-908: Added query to get silenced alert and only display non silenced alert

### PR DESCRIPTION
**Pull request by OlivierCazade**
The banner will only appear after both query have been done and only non silenced alert will appear.

### CODE REVIEW

File: web/src/components/alerts/fetcher.tsx [hunk 1]
```diff
@@ -38,10 +39,34 @@ export const AlertFetcher: React.FC<AlertFetcherProps> = ({ children }) => {
...
+          result
+            .filter(rule => rule.status.state == 'active')
+            .map(rule => {
+              const matcher = rule.matchers.find(m => m.name == 'alertname');
+              return typeof matcher !== 'undefined' ? matcher.value : '';
+            })
+        );
+      })
+      .catch(() => {
+        console.log('Could not get alerts');
```
Comment (jpinsonneau) [inline web/src/components/alerts/fetcher.tsx:57]: nit:

```suggestion
        console.log('Could not get silenced alerts');
```
Should we also set silenced alertes to `[]` to show all alerts in that case ?

File: web/src/components/alerts/fetcher.tsx [hunk 2]
```diff
@@ -38,10 +39,34 @@ export const AlertFetcher: React.FC<AlertFetcherProps> = ({ children }) => {
       });
     return;
   }, []);
+
+  React.useEffect(() => {
+    getSilencedAlerts()
+      .then(result => {
+        setSilencedAlerts(
+          /* result.filter(rule => rule.status.state == 'active').map(rule => rule.id) */
```
Comment (jpinsonneau) [inline web/src/components/alerts/fetcher.tsx:47]: nit:

```suggestion
```

**Review by jpinsonneau — APPROVED** [review]
Small nit comments, LGTM in terms of code

File: web/src/components/alerts/fetcher.tsx (hunk 1 above)
Comment (OlivierCazade) [reply web/src/components/alerts/fetcher.tsx]: Done, thanks

File: web/src/components/alerts/fetcher.tsx (hunk 2 above)
Comment (OlivierCazade) [reply web/src/components/alerts/fetcher.tsx]: Done!

**Review by jpinsonneau — APPROVED** [review]
LGTM, thanks @OlivierCazade !

**Pull request merged**